src/components/Tabs/Tabs/AnimatedTabs.tsx

```diff
@@ -22,6 +23,7 @@ export const AnimatedTabs: FC<TabsProps> = React.forwardRef(
         const { currentActiveTab } = useTabs();
         const tabClassName: string = mergeClasses([
             styles.tabWrap,
+            { [styles.underlined]: underlined && variant !== TabVariant.pill },
```

Comment:
may want to verify that a double negation isn't needed here, e.g. undefined in the classList.

Reply:
not seeing any evidence of undefined in the classList. I believe the defaulting to false ensures we are working with a proper boolean value